test/templates/default.tpl.js

```diff
@@ -2,16 +2,20 @@
 const { Kuzzle } = require('kuzzle-sdk');
 
 // instantiate a Kuzzle client
-const kuzzle = new Kuzzle('websocket', { host: 'kuzzle', autoReconnect: false });
+const kuzzle = new Kuzzle('websocket', {
+  host: 'kuzzle',
+  autoReconnect: false
+});
 
 // add a listener to detect any connection problems
 kuzzle.on('networkError', error => {
   console.error('Network Error:' + error);
-})
+});
 
-kuzzle.connect()
+kuzzle
+  .connect()
   .then(() => {
-    return [snippet-code]
+    return [snippet - code];
```

Comment:
This is a custom tag, I don't think it will still working with space around the `-`